tasks/test-project/test-project

```diff
@@ -164,6 +196,29 @@ const globalTasks = () =>
           )
         },
       },
+      {
+        title: 'Updating ports in redwood.toml...',
+        task: () => {
+          const REDWOOD_TOML_PATH = path.join(
+            OUTPUT_PROJECT_PATH,
+            'redwood.toml'
+          )
+          const redwoodToml = fs.readFileSync(REDWOOD_TOML_PATH).toString()
+          let newRedwoodToml = redwoodToml
+
+          newRedwoodToml = newRedwoodToml.replace(
+            /\port = 8910/,
+            'port = "${WEB_DEV_PORT:8910}"'
+          )
+
+          newRedwoodToml = newRedwoodToml.replace(
+            /\port = 8911/,
+            'port = "${API_DEV_PORT:8911}"'
+          )
```

Comment:
Could this go in the Playwright fixtures instead of here, where it will be included in every built Test-project (for both CI and local dev)?